sympy/concrete/summations.py

```diff
@@ -429,6 +433,15 @@ def is_convergent(self):
 
         order = O(sequence_term, (sym, S.Infinity))
 
+        ### ----------- ratio test ---------------- ###
+        next_sequence_term = sequence_term.xreplace({sym: sym + 1})
+        ratio = simplify(next_sequence_term/sequence_term)
```

Comment:
simplify should be avoided in favor of more specific operations, as [its own docstring](http://docs.sympy.org/latest/modules/simplify/simplify.html#sympy.simplify.simplify.simplify) says. Here you want to cancel powers of n (that's done by `powsimp`) and the factorials (with `combsimp`). So, use `combsimp(powsimp(next_sequence_term/sequence_term))`